core/core-impl/src/test/java/ai/timefold/solver/core/impl/score/stream/ConsecutiveSetTreeTest.java

```diff
@@ -79,9 +101,9 @@ void testConsecutiveNumbers() {
         tree.add(atomic(8), 8);
 
         IterableList<Sequence<AtomicInteger, Integer>> sequenceList = new IterableList<>(tree.getConsecutiveSequences());
-        assertThat(sequenceList).hasSize(2);
+        Assertions.assertThat(sequenceList).hasSize(2);
```

Comment:
Why use `Assertions.assertThat` when `import static org.assertj.core.api.Assertions.assertThat;` is in the file?